tmt/frameworks/beakerlib.py

```diff
@@ -11,11 +11,21 @@
 from tmt.utils import Path
 
 if TYPE_CHECKING:
+    from tmt.base import Dependency, Test
     from tmt.steps.execute import TestInvocation
 
 
 @provides_framework('beakerlib')
 class Beakerlib(TestFramework):
+    @classmethod
+    def get_requirements(
+            cls,
+            test: 'Test',
+            logger: tmt.log.Logger) -> list['Dependency']:
+        from tmt.base import DependencySimple
```

Comment:
Is it necessary to import in the method and not on the top with the rest of imports?

Reply:
I think this is a case of circular imports, `tmt.base` definitely imports `tmt.frameworks` because of `Test.test_framework`, and `tmt.frameworks` then needs to instantiate `tmt.base.DependencySimple`... But I can add a comment pointing it out.

Reply:
Added in c16a8ab1.